core/pkg/storage/docker/schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE STATUS AS ENUM('UNKNOWN', 'PENDING', 'PARSING_FAILED', 'TRANSPORT_FAILED');
```

Comment:
You forgot `SUCCESS`  😄 

Also, let's have only one failure enum. Details could be stored as any text later. 


PENDING - in progress of detection, no status yet, but acknowledged 
UNKNOWN - request is completely unknown to server
SUCCESS - request was successfully processed and contains some verdict from compute 
FAILURE - anything went wrong during request processing on backend side.